src/sysfiles.c

```diff
@@ -2062,7 +2062,7 @@ static void initreadline(void)
   /* this should pipe signals through to GAP  */
   rl_already_prompted = 1 ;
 
-  rl_catch_signals = 0;
+  rl_catch_signals = 1;
```

Comment:
So I am slightly concerned about other consequences this change might have with respect to signal handling, and interaction with other system; in particular for the integration of GAP into Julia.  I also don't really understand why/how this fixes the issue you are facing?